scripts/release/debian/deb_install.sh

```diff
@@ -81,13 +82,17 @@ setup() {
             exit 1
         fi
     fi
-    echo "deb [arch=$(dpkg --print-architecture)] https://packages.microsoft.com/repos/azure-cli/ ${CLI_REPO} main" \
-        > /etc/apt/sources.list.d/azure-cli.list
```

Comment:
We also need to clean `/etc/apt/sources.list.d/azure-cli.list`

Reply:
yeah, i'll add a test to remove `azure-cli.list` if it exists.

Reply:
done